Autocoders/Python/test/enum_xml/main.cpp

```diff
@@ -17,25 +17,21 @@
 
 using namespace std;
 
-// Component instance pointers
-Example::ExampleEnumImpl* inst1 = nullptr;
-Example::ExampleEnumImpl* inst2 = nullptr;
+// Instantiate the inst1 and inst2 components
```

Comment:
Why was this changed?  General cleanup?

Reply:
Yes, I believe we used this test as the basis for a new test, noticed that this test used an older style for creating test components, and modernized both tests.